### DISCUSSION TITLE
HomeAssistant Integration Cam Entity not available

### DISCUSSION THREAD

**Discussion by dnwalker68** [blakeblackshear/frigate#4858] [Ask A Question]
Today I installed the current version of Frigate on a current HomeAssistant system. The system starts and also writes data to the MQTT server. The camera image can be seen and so far everything works very well.
After that, I wanted to include the HomeAssistant integration. The problem here is that only the Frigate entity works. The camera's entities are all in the "unavailable" state. I tried the installation several times but the error persists.

I can not find the error and hope you can give me a solution.
Thanks and regards
dieter

frigate.yml:

mqtt:
  host: 192.168.1.88
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: mqtt-user
  password: xxxxxxxx
  stats_interval: 10

detectors:
  cpu1:
    type: cpu
    num_threads: 3

cameras:
  door:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.157:554/12
          roles:
            - detect
            - rtmp
            
    rtmp:
      enabled: False # <-- RTMP should be disabled if your stream is not H264
    detect:
      width: 800 # <---- update for your camera's resolution
      height: 600 # <---- update for your camera's resolution

![MQTT](https://user-images.githubusercontent.com/99841809/210266515-93755b29-b1dc-4c04-b084-6d470efa58d8.JPG)
![Entity](https://user-images.githubusercontent.com/99841809/210266638-d2f9708d-5325-4f18-a13d-4ce9fb10e28c.JPG)

**Comment by NickM-27** [comment]
Have you setup the MQTT integration in Home Assistant? as the Frigate integration README explains that is required
